_projects/blocmetrics.md

```diff
@@ -1,33 +1,16 @@
 ---
-layout: page
+layout: projects
 title: Blocmetrics
-thumbnail-path: "/images/analytics.png"
+src-code: https://github.com/javierforero/Blocmetrics
+image-url: /images/blocitoff.png
+heroku-url: https://blocmetrics-javi.herokuapp.com/
 short-description: Blocmetrics is an analytics site that allows you to track events on your web apps.
 ---
 
-**Summary**
 
-Blocmetrics is a web app that allows you to track events from other sites via an API and displays them in a graphic form.  
+Snapshot
+============
 
-**Explanation**
+Blocmetrics is a web app that allows you to track events from other sites via an API and displays them in a graphic form.   This was built using Ruby on Rails.
 
-This web application was made to help users track events such as clicks or impressions on their websites.
-
-**Problem**
-
-Data plays a huge role in today’s market. Knowing what’s trending and what is not is important for a business to be a able to succeed in ecommerce.
-
-
-**Solution**
-
-Blocmetrics solves this issue by allowing users to track as many events from their websites. Blocmetrics then collects this data and displays them graphically providing instant insights.
-
-**Results**
-
-What worked really well for Blocmetrics was having two charts. A pie chart that displays the frequency of an event and a line chart that displays when events were created. Although this is very useful information, most users want to be able to choose how they want to see the data and also have a way to export it as a csv file. These are definitely improvements that will be made available.
-
-![alt text](../img/blocmetrics.png)
- 
-**Conclusion**
-
-Blocmetrics is a powerful application that is easy to integrate with any web application.
+This web application was made to help users track events such as clicks or impressions on their websites. Blocmetrics allows users to track as many events from their websites and displays them graphically providing instant insights.
```

Comment:
do you want to mention anything about challenges associated with accepting clientside API requests from other applications?